packages/interface/src/store/actions.js

```diff
@@ -181,3 +181,28 @@ export function setFeatureDefaults( scope, defaults ) {
 		registry.dispatch( preferencesStore ).setDefaults( scope, defaults );
 	};
 }
+
+/**
+ * Returns an action object used in signalling that the user opened a modal.
+ *
+ * @param {string} name A string that uniquely identifies the modal.
+ *
+ * @return {Object} Action object.
+ */
+export function openModal( name ) {
```

Comment:
Now should we make the edit-post actions and selectors work as proxies to the "interface" store for backward compatibility?

Reply:
You mean like that:
```
export const closeModal =
	() =>
	( { registry } ) => {
		deprecated( "select( 'core/edit-post' ).closeModal()", {
			since: '6.3',
			alternative: "select( 'core/interface').closeModal()",
		} );
		return registry.dispatch( interfaceStore ).closeModal();
	};
```
right?

I'm writing this code right now.

Reply:
👍 Yes, this works for me